Eatery/Controllers/MealTableViewController.swift

```diff
@@ -45,6 +51,8 @@ class MealTableViewController: UITableViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        dateFormatter.dateFormat = "YYYY-MM-dd"
```

Comment:
(If dateFormatter should be in both this view controller and eateries VC, then this should be closer to the declaration of dateFormatter as well.)